src/fpjs-provider.tsx

```diff
@@ -37,18 +39,26 @@ export function FpjsProvider<TExtended extends boolean>({
   cacheLocation,
   loadOptions,
 }: PropsWithChildren<FpjsProviderOptions>) {
+  const [env, setEnv] = useState<EnvDetails | undefined>()
+
   const clientOptions = useMemo(() => {
+    const integrationInfo = [...(loadOptions.integrationInfo || []), `fingerprintjs-pro-react/${packageInfo.version}`]
+
+    if (env) {
+      integrationInfo.push(env.version ? `${env.name}/${env.version}` : env.name)
```

Comment:
It is not an integration, so the idea was to add it as the third param, like `fingerprintjs-pro-react/${packageInfo.version}/${env.name}`

Reply:
Changed!